src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mcfooter/MixedContentFooter.java

```diff
@@ -76,4 +109,48 @@ public int countArticleCards () {
     return articleLinks.size();
   }
 
+  public int countArticlesInExploreCard() {
+    wait.forElementVisible(exploreWikisCard);
+    return exploreWikisList.size();
+  }
+
+  public void clickOnViewAllLinkInDiscussions() {
+    wait.forElementVisible(viewAllLink).click();
```

Comment:
@qaga should be `wait.forElementClickable(..)` when you click on the element

Reply:
done